cmd/init.go

```diff
@@ -49,10 +48,10 @@ func runInit(cmd *cobra.Command, args []string) error {
 }
 
 type initConfig struct {
-	// Name of the service in DNS-compatible format (ex myfunc.example.com)
+	// Name of the function.
```

Comment:
Minor nitpick about the comment:  that should probably be `Function`.  In order to avoid confusion with the built-in function primitive of most languages, to name our functions as proper nouns; thus capitalized.